docs/TROUBLE.md

```diff
@@ -0,0 +1,162 @@
+
+- Check your serial port wiring to ensure you have connected rx to tx and tx to tx and ground to ground.
+
+- Use the mLRS LUA script or another configuration method to check your [mLRS Rx setup](CRSF.md#mlrs-rx-module-setup) to be sure MAVLink is selected for the Rx Ser Link Mode and that the baud rate is correct.
+
+- Use a USB connected ground control station to check your ArduPilot SERIALx_* [configuration](CRSF.md#ardupilot-setup) to be sure the correct SERIALx\_PROTOCOL is set to MAVLink2 and that the SERIALx\_BAUD matches what you set the Rx to use.
+
+- If it is still not working, use a volt meter or oscilloscope to verify that both serial port wires between the Rx and flight controller have a voltage with respect to ground that is constantly changing.  If either wire has no changing signal, you have probably still missed something above.
+
+## 5. Confirm MAVLink message flow from Tx module to ground station
+
+Start your ground station software and connect to your Tx module by whatever method you have chosen.  You should see or hear the flight mode announced.  If parameter download also completes successfully, you can skip the next two topics
+
+This is a complex topic since there are so many ways to connect a Tx module to a ground station.
+
+- Use the [mLRS LUA script](LUA.md#usage) or [CLI](CLI.md) to check that Tx Ser Dest is correct and that Tx Ser Baudrate matches your wireless bridge or ground control setting.
+
+- If the connection is via a serial port, double check that rx is connected to tx, tx to rx, and ground to ground.
+
+- If your Tx module is the [R9M](FRSKY_R9.md#r9m-tx-module), don't forget to account for the inverted serial port and set dip switch 1 to on before powering up.
+
+- If the ground station runs on a mobile device and is connected via USB, an OTG adapter may be required.
+
+- If you want to connect via Bluetooth to the original [MatekSys Tx Module](MATEKSYS.md), don't forget to enable the HC-04 by moving all three [dip switches](https://www.mateksys.com/?portfolio=mr900-30-tx#tab-id-2) to the on position.
+
+- If the connection is via Bluetooth, the devices need to be paired first.
+
+- If the connection is via UDP over WiFi, connect to WiFi access point before starting the ground station.
+
+## 6. Confirm MAVLink message flow from Rx to flight controller
+
+Use the mLRS LUA script or another configuration method to temporarily set [Rx Snd RcChannel](PARAMETERS.md#rx-snd-rcchannel) to RC\_CHANNELS\_OVERRIDE.  If you use the mLRS LUA script there is no need to save; Just press the RTN button after selecting RC\_CHANNELS\_OVERRIDE.
+
+Connect a ground station via USB directly to your flight controller and view the MAVLink messages.
```

Comment:
maybe some enclosing "" could help

Reply:
Not sure where you could want quotes on this line.

Reply:
me neither LOL